core/src/main/java/io/lionweb/lioncore/java/model/impl/DynamicClassifierInstance.java

```diff
@@ -1,13 +1,16 @@
 package io.lionweb.lioncore.java.model.impl;
 
 import io.lionweb.lioncore.java.language.*;
-import io.lionweb.lioncore.java.model.*;
+import io.lionweb.lioncore.java.model.ClassifierInstance;
+import io.lionweb.lioncore.java.model.HasSettableParent;
+import io.lionweb.lioncore.java.model.Node;
+import io.lionweb.lioncore.java.model.ReferenceValue;
 import java.util.*;
 import java.util.stream.Collectors;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-public abstract class DynamicClassifierInstance<T extends Classifier<T>>
+public abstract class DynamicClassifierInstance<T extends Classifier<T>> extends AnnotatedNode<T>
```

Comment:
Aren't nodes a subset of ClassifierInstances? If so a ClassifierInstance extending a Node seems strange